lib/SIL/IR/SILBuilder.cpp

```diff
@@ -174,7 +174,8 @@ SILBuilder::createUncheckedReinterpretCast(SILLocation Loc, SILValue Op,
 
 // Create the appropriate cast instruction based on result type.
 SingleValueInstruction *
-SILBuilder::createUncheckedBitCast(SILLocation Loc, SILValue Op, SILType Ty) {
+SILBuilder::createUncheckedForwardingCast(SILLocation Loc, SILValue Op,
+                                          SILType Ty) {
   // Without ownership, delegate to unchecked reinterpret cast.
   if (!hasOwnership())
     return createUncheckedReinterpretCast(Loc, Op, Ty);
```

Comment:
This can call UncheckedBitWiseCast::create, so not really forwarding cast ? : )

Reply:
Without ownership, the forwarding cast degenerates into a bit-wise cast. I can explain that in comments.